src/Common/src/Interop/Windows/winhttp/Interop.winhttp_types.cs

```diff
@@ -289,26 +279,12 @@ public struct WINHTTP_ASYNC_RESULT
         public const uint API_WRITE_DATA = 4;
         public const uint API_SEND_REQUEST = 5;
 
-        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
-        public struct WINHTTP_WEB_SOCKET_ASYNC_RESULT
-        {
-            public WINHTTP_ASYNC_RESULT AsyncResult;
-            public WINHTTP_WEB_SOCKET_OPERATION Operation;
-        }
-
         public enum WINHTTP_WEB_SOCKET_OPERATION
```

Comment:
This enum type could probably be removed as well.  We don't use WinHTTP for WebSocket anymore (we use the managed client on top of SocketsHttpHandler.